### PR TITLE
0.3.1

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
Blocked by https://github.com/aimhubio/aimrocks/issues/10
Blocking for https://github.com/conda-forge/aim-feedstock/pull/19

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/aimhubio/aimrocks/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c30ff542ba95cd615616cec07061f33e8385ebd131801fc2cdaf83010241a469
+  # 0.3.1: waiting for a proper git tag
+  url: https://github.com/aimhubio/aimrocks/archive/812f15af21b5e137159f8b5c620b7b6b65bee8b4.tar.gz
+  sha256: b9289febb49ca31bec6300e8d71c2551b5b7781d6274471481733eb23d845f77
```
Comment (hadim) [inline recipe/meta.yaml]: Waiting for a git tag or sdist pypi package for the source.

**Comment by hadim** [comment]
Eventually wait for https://github.com/aimhubio/aimrocks/pull/26 to be merged.

**Comment by hadim** [comment]
@gorarakelyan this PR is required to be able to bump the aim 3.16 conda package. The only thing preventing from merging here is having a git tag or a sdist upload to pypi of aimrocks 0.3.1.

**Comment by hadim** [comment]
@gorarakelyan do you think you can help move this forward?

**Comment by arkkln** [comment]
@hadim sorry for a late reply, the thread was somehow lost in my notifications. 
Just pushed v0.3.1 tag for aimrocks. Is it enough to proceed further?

**Comment by hadim** [comment]
> @hadim sorry for a late reply, the thread was somehow lost in my notifications. Just pushed v0.3.1 tag for aimrocks. Is it enough to proceed further?

Thanks. I think it should be good now.

**Pull request merged**